app/assets/stylesheets/_chapters-show.scss

```diff
@@ -87,6 +87,78 @@ body.chapters-show {
       }
     }
   }
+
+  .owl-dots-wrapper {
+    margin-top: 20px;
+
+    .owl-dots{
+      float: left;
+      width: 20%;
+      margin-left: 40%;
+      margin-right: 40%;
```

Comment:
Can you explain why this is 20% with 40% margins instead of just being 100% with no margins? I ask specifically because on smaller screens (looking on my iphone 5S) this only supports about 7 dots before wrapping. Before making the change I just wanted to know if there was a reason for this - otherwise we might as just go 100%.

![img_5143](https://user-images.githubusercontent.com/2909/45267585-316c8380-b43d-11e8-9617-a498291b3001.PNG)

Reply:
I believe it was just the OwlCarousel default :) Had few images locally so missed that bug. I'll make the width 100% on mobiles, and width some meaningful paddings in bigger screens.

Reply:
Looking at it, it looks like a smaller size might actually help aesthetically, since having a 100% width gives us 100% width worth of dots. But I'm not sure there's anything we can do about this, so I'll let it go.
![fullsizerender](https://user-images.githubusercontent.com/2909/48572443-71396680-e8d7-11e8-9672-0c2811fc239d.jpg)

Reply:
👍